npm_and_yarn/lib/dependabot/npm_and_yarn/file_updater/npm_lockfile_updater.rb

```diff
@@ -442,6 +444,18 @@ def git_dependencies_to_lock
           @git_dependencies_to_lock
         end
 
+        def lock_deps_with_latest_reqs(content)
+          json = JSON.parse(content)
+          NpmAndYarn::FileParser::DEPENDENCY_TYPES.each do |type|
+            json.fetch(type, {}).each do |nm, _|
```

Comment:
Lets rename `_` to `requirement` like the yarn one.

You'd usually name a variable `_` when it isn't used but required for the method call.